shared/js/initApp.js

```diff
@@ -33,7 +33,7 @@ window.apps = {
         if (window.dashboard.isChrome34) {
           chrome34Fix.fixup();
         }
-        if (appOptions.level.projectTemplateLevelName) {
+        if (appOptions.level.projectTemplateLevelName || appOptions.app === 'applab') {
```

Comment:
Note to future selves: when we decide to enable this for Pixelation, we can just change this to `appOptions.isChannelBacked`